### PR TITLE
[Doc] Modify documentation and add troubleshooting guidance

### PR DESCRIPTION
**Pull request by vinli-cn**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Modify documentation
**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: docs/csi-debug.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+csi-azurelustre-controller-56bfddd689-dh5tk       3/3     Running   0          35s     10.240.0.19    k8s-agentpool-22533604-0
+csi-azurelustre-controller-56bfddd689-sl4ll       3/3     Running   0          35s     10.240.0.23    k8s-agentpool-22533604-1
+</pre>
+
+&nbsp;
+
+- Get csi driver logs
+
+```console
+$ kubectl logs csi-azurelustre-controller-56bfddd689-dh5tk -c azurelustre -n kube-system > csi-lustre-controller.log
```
Comment (eridai) [inline docs/csi-debug.md:26]: What is the log retention policy for kubernetes Pod?

File: docs/csi-debug.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
+## CSI driver troubleshooting guide
+
+&nbsp;
+
+### Case#1: volume create/delete issue
```
Comment (eridai) [inline docs/csi-debug.md:5]: How CSI user can identify it was a create/delete issue or mount/unmount issue? Will the kubctl returns the error?

File: docs/csi-debug.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+- Update daemonset deployment
+
+```console
+$ kubectl edit ds csi-azurelustre-node -n kube-system
+```
+
+&nbsp;
+
+- Change lustre image config
```
Comment (eridai) [inline docs/csi-debug.md:98]: We are referring to Lustre CSI docker image, right?

File: docs/csi-dev.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+```console
+$ csc node unpublish --endpoint $endpoint --target-path $target_path $volname
+```
+
+&nbsp;
+
+#### 5. Delete azurelustre volume
+
+```console
+$ csc controller del --endpoint $endpoint CSIVolumeID
```
Comment (eridai) [inline docs/csi-dev.md:136]: Is CSIVolumeID something user need to provide here?

File: docs/csi-debug.md (hunk 1 above)
Comment (vinli-cn) [reply docs/csi-debug.md:26]: {
  "live-restore": true,
  "log-driver": "json-file",
  "log-opts":  {
     "max-size": "50m",
     "max-file": "5"
  }
}

File: docs/csi-debug.md (hunk 2 above)
Comment (fuxiuyin) [reply docs/csi-debug.md:5]: If it was a create/delete issue. User will observe PV cannot be created/deleted. And POD is not scheduled. If it was a mount/umount issue. User will observe POD fail to start. And POD is scheduled.

File: docs/csi-debug.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+&nbsp;
+
+- Get csi driver logs
+
+```console
+$ kubectl logs csi-azurelustre-controller-56bfddd689-dh5tk -c azurelustre -n kube-system > csi-lustre-controller.log
+```
+
+> note: there could be multiple controller pods, logs can be taken from all of them simultaneously, also with `follow` (realtime) mode
```
Comment (fuxiuyin) [inline docs/csi-debug.md:29]: Perhaps we could optimize all this work into a single kubectl logs command like:
kubectl logs -nkube-system -lapp=csi-azurelustre-controller -cazurelustre --tail=-1 --prefix >csi-lustre-controllers.log

File: docs/csi-dev.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+```console
+$ make verify
+```
+
+&nbsp;
+
+- If there is config file changed under `charts` directory, run following command to update chart file.
+
+```console
+$ helm package charts/latest/azurelustre-csi-driver -d charts/latest/
```
Comment (fuxiuyin) [inline docs/csi-dev.md:38]: I don't think we've tested helm yet.

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**